tutorials/30_File_Type_Preprocessing_Index_Pipeline.ipynb

```diff
@@ -0,0 +1,1328 @@
+{
```

Comment:
<p>If you decide to update the mime_types list, don't forget to update the connection here.</p><p><code>preprocessing_pipeline.connect("file_type_router.unclassified", "markdown_converter.sources")</code> -&gt; <code>preprocessing_pipeline.connect("file_type_router.text/markdown", "markdown_converter.sources") </code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/274/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>oh thanks, I definitely would not have caught that!</p>